src/pykeen/losses.py

```diff
@@ -86,26 +175,79 @@ class BCEWithLogitsLoss(PointwiseLoss, nn.BCEWithLogitsLoss):
 
         This loss is not well-suited for translational distance models because these models produce
         a negative distance as score and cannot produce positive model outputs.
+
+    .. seealso:: :class:`torch.nn.BCEWithLogitsLoss`
     """
 
     synonyms = {'Negative Log Likelihood Loss'}
 
+    def forward(
+        self,
+        scores: torch.FloatTensor,
+        labels: torch.FloatTensor,
+    ) -> torch.FloatTensor:  # noqa: D102
+        return functional.binary_cross_entropy_with_logits(scores, labels, reduction=self.reduction)
+
 
-class MSELoss(PointwiseLoss, nn.MSELoss):
-    """A wrapper around the PyTorch mean square error loss."""
+class MSELoss(PointwiseLoss):
+    """A wrapper around the PyTorch mean square error loss.
+
+    .. seealso:: :class:`torch.nn.MSELoss`
+    """
 
     synonyms = {'Mean Square Error Loss', 'Mean Squared Error Loss'}
 
+    def forward(
+        self,
+        scores: torch.FloatTensor,
+        labels: torch.FloatTensor,
+    ) -> torch.FloatTensor:  # noqa: D102
+        assert self.validate_labels(labels=labels)
+        return functional.mse_loss(scores, labels, reduction=self.reduction)
+
 
-class MarginRankingLoss(nn.MarginRankingLoss, PairwiseLoss):
-    """A wrapper around the PyTorch margin ranking loss."""
+class MarginRankingLoss(PairwiseLoss):
+    """A wrapper around the PyTorch margin ranking loss.
+
+    .. seealso:: :class:`torch.nn.MarginRankingLoss`
+    """
 
     synonyms = {"Pairwise Hinge Loss"}
 
     hpo_default: ClassVar[Mapping[str, Any]] = dict(
         margin=dict(type=int, low=0, high=3, q=1),
     )
 
+    def __init__(
+        self,
+        margin: float = 1.0,
+        margin_activation: Callable[[torch.FloatTensor], torch.FloatTensor] = functional.relu,
+        reduction: str = 'mean',
+    ):
+        """Initialize the margin loss instance.
+
+        :param margin:
+            The margin by which positive and negative scores should be apart.
+        :param margin_activation:
+            A margin activation. Defaults to relu, i.e. f(x) = max(0, x), which is the default "margin loss". Using e.g.
+            softplus leads to a "soft-margin" formulation, as e.g. discussed here https://arxiv.org/abs/1703.07737
```

Comment:
rather than making the user decide based on a parameter, do you think it would be better to have two subclasses "Margin loss' and "soft margin loss" where the activation is fixed?

Reply:
Yes, would be fine as well.

Reply:
Will do this in the next PR. I also have to rectify the situation where specific checks are made for NSSAL and MRL